docs/specs/sdk-settings-actions.md

```diff
@@ -0,0 +1,292 @@
+# SDK - Settings Actions
```

Comment:
I prefer this design. It is much easier to understand. No need to understand what is an Authgear authenticator.